immobilienscout.gemspec

```diff
@@ -1,17 +1,36 @@
-Gem::Specification.new do |s|
-  s.name         = 'immobilienscout'
-  s.version      = '0.0.3'
-  s.date         = '2019-03-11'
-  s.summary      = 'A ruby gem to handle Immobilienscout API'
-  s.description  = 'This gem provides HTTP endpoints to connect your app with Immobilienscout API'
-  s.authors      = ['Angela Guette', 'Wiebke Frey', 'Laura Garcia']
-  s.email        = 'backend-team@homeday.de'
-  s.files        = Dir.glob('lib/**/*') + ['README.md']
-  s.homepage     = 'https://github.com/homeday-de/immobilienscout'
-  s.license      = 'MIT'
-  s.require_path = 'lib'
-  s.add_dependency 'json'
-  s.add_dependency 'multipart-post'
-  s.add_dependency 'activesupport'
-  s.add_dependency 'webmock'
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'immobilienscout/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'immobilienscout'
+  spec.version       = Immobilienscout::VERSION
+  spec.authors       = ['Angela Guette', 'Wiebke Frey', 'Laura Garcia']
+  spec.email         = ['backend-team@homeday.de']
+
+  spec.summary       = 'A ruby gem to handle Immobilienscout API'
+  spec.description   = 'This gem provides HTTP endpoints to connect your app with Immobilienscout API'
```

Comment:
It does not provide HTTP _endpoints_, it provides a HTTP _client_.

Reply:
Right!